sdk/monitor/monitor-opentelemetry-exporter/src/utils/spanUtils.ts

```diff
@@ -202,6 +215,13 @@ function createDependencyData(span: ReadableSpan): RemoteDependencyData {
   const rpcSystem = span.attributes[SemanticAttributes.RPC_SYSTEM];
   // HTTP Dependency
   if (httpMethod) {
+    const httpUrl = span.attributes[SemanticAttributes.HTTP_URL];
+    if (httpUrl) {
+      try {
+        let dependencyUrl = new URL(String(httpUrl));
+        remoteDependencyData.name = `${httpMethod} ${dependencyUrl.pathname}`;
```

Comment:
Comment about default `/` applies here as well.

Reply:
Is added by URL object, there must be a test in this PR where this is validated